lib/jekyll/assets/proxies/magick.rb

```diff
@@ -9,10 +9,10 @@
   presets = %W(@2x @4x @1/2 @1/3 @2/3 @1/4 @2/4 @3/4
     @double @quadruple @half @one-third @two-thirds @one-fourth
       @two-fourths @three-fourths)
+  ARGS=args
+  PRESETS=presets
```

Comment:
I've removed `PRESETS` since it's not referenced anywhere, though how to deal with `ARGS` - I'm not sure, requesting guidance